### PR TITLE
Html semantics

### PR DESCRIPTION
**Pull request by Hannahdlewis**
added header code to all 3 html files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,34 @@
     <link rel="shortcut icon" type="image/jpg" href="assets/images/rotten_trailers_favicon.png"/>
     <title>Rotten Trailers</title>
 </head>
+<header>
+    <div id=logo>
+        <img src="Rottentrailerslogo will go here" alt="Rottern Trailers Logo"/>
+    </div>
+         <nav>
+            <button type="button" onclick="alert('it works!')">Movies</button>
```
Comment (Hannahdlewis) [inline index.html]: please make this purple

Comment (jeffcela) [inline index.html]: need to add in the id: "btn btn-movies"

File: index.html [hunk 2]
```diff
@@ -7,6 +7,34 @@
...
+            <button type="button" onclick="alert('it works!')">TV</button>
```
Comment (jeffcela) [inline index.html]: need to add in the id: "btn btn-tv"

File: index.html [hunk 3]
```diff
@@ -7,6 +7,34 @@
...
+            <button type="button" onclick="alert('it works!')">Watch Later</button>
```
Comment (jeffcela) [inline index.html]: need to add in the id: "btn btn-watchLater"

File: index.html [hunk 4]
```diff
@@ -7,6 +7,34 @@
     <link rel="shortcut icon" type="image/jpg" href="assets/images/rotten_trailers_favicon.png"/>
     <title>Rotten Trailers</title>
 </head>
+<header>
+    <div id=logo>
+        <img src="Rottentrailerslogo will go here" alt="Rottern Trailers Logo"/>
```
Comment (jeffcela) [inline index.html]: I think the src should be changed to since it is a local path href: "assets/images/rotten_trailers_logo-01.png"

**Review by jeffcela — COMMENTED** [review]
I have left comments that should be applied to ALL .html files

File: index.html (hunk 3 above)
Comment (Hannahdlewis) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (Hannahdlewis) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (Hannahdlewis) [reply index.html]: done

**Review by jeffcela — APPROVED** [review]

**Review by ptriv1 — APPROVED** [review]

**Pull request merged**